src/font.ts

```diff
@@ -326,16 +324,34 @@ export class Font {
    * Alternative: https://cdn.jsdelivr.net/npm/vexflow-fonts@1.0.3/
    * Or you can use your own host.
    */
-  static WEB_FONT_HOST = 'https://unpkg.com/vexflow-fonts@1.0.3/';
+  static WEB_FONT_HOST = 'https://unpkg.com/vexflow-fonts@1.0.5/';
 
   /**
    * These font files will be loaded from the CDN specified by `Font.WEB_FONT_HOST` when
    * `await Font.loadWebFonts()` is called. Customize this field to specify a different
    * set of fonts to load. See: `Font.loadWebFonts()`.
    */
   static WEB_FONT_FILES: Record<string /* fontName */, string /* fontPath */> = {
-    'Roboto Slab': 'robotoslab/RobotoSlab-Medium_2.001.woff',
-    PetalumaScript: 'petaluma/PetalumaScript_1.10_FS.woff',
+    Bravura: 'bravura/Bravura_1.392.woff2',
+    BravuraText: 'bravura/BravuraText_1.393.woff2',
+    GonvilleSmufl: 'gonvillesmufl/GonvilleSmufl_1.100.woff2',
+    Gootville: 'gootville/Gootville_1.3.woff2',
+    GootvilleText: 'gootville/GootvilleText_1.2.woff2',
+    Leland: 'leland/Leland_0.75.woff2',
+    LelandText: 'leland/LelandText_0.75.woff2',
```

Comment:
Note how the two deleted lines ended in *.woff.

There is no need to specify the woff2 file extension, because the loadWebFonts() function will automatically add the woff2 extension before providing it to the FontFace API.

Reply:
I decided to go explicitely to woff2 because there was a lot of discussion on network traffic and total length of files.